src/Mvc/Mvc.Formatters.Json/test/Microsoft.AspNetCore.Mvc.Formatters.Json.Test.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is there a reason why we need a `Formatters.Json` at all? I don't think that the reasons why we did this in the past are really relevant anymore.